src/webextension/list/index.js

```diff
@@ -0,0 +1,31 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
It feels odd to have an `index.js` that's not an export of public things in `/list`.